src/invert/parderiv/impls/cyclic/cyclic.cxx

```diff
@@ -147,11 +140,11 @@ const Field3D InvertParCR::solve(const Field3D &f) {
         
         //           const     d2dy2        d2dydz             d2dz2           ddy
         //           -----     -----        ------             -----           ---
-	a[k][y+y0] =            bcoef - 0.5*Im*kwave*ccoef                  -0.5*ecoef;
-	b[k][y+y0] = acoef - 2.*bcoef                     - SQ(kwave)*dcoef;
-	c[k][y+y0] =            bcoef + 0.5*Im*kwave*ccoef                  +0.5*ecoef;
-	
-	rhsk[k][y+y0] = rhs[y+y0][k]; // Transpose
+        a(k, y + y0) = bcoef - 0.5 * Im * kwave * ccoef - 0.5 * ecoef;
```

Comment:
The whitespace changes have made the preceding comment not very useful. Should we either preserve the whitespace, or delete/modify the comment?

Reply:
I'll revert the whitespace.